selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromNodeStackTest.java

```diff
@@ -86,7 +87,7 @@ public void checkWorkspaceCreationFromNodeStack() {
     projectExplorer.waitProjectInitialization(WEB_NODE_JS_PROJECT);
   }
 
-  @Test(priority = 1)
+  @Test(priority = 1, groups = UNDER_REPAIR)
```

Comment:
It's expected that test under repair points to issue which describe **permanent** test failure, in try-catch block around failing test command.